lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
Minor: what about renaming `payment_partial_name` into `payment_type`? And getting the partial name using something like the code below in the calculator?
```ruby
def services_payment_partial_name
  ceremony_country_data['payment_type']
end
```
Not sure about talking about partials in the config file?